modules/react-components/src/components/avatar/user-avatar.tsx

```diff
@@ -85,6 +88,10 @@ export const UserAvatar: FunctionComponent<UserAvatarPropsInterface> = (
     const [ userImage, setUserImage ] = useState(null);
     const [ showPopup, setShowPopup ] = useState(false);
 
+    // To check if the size is a valid Semantic UI size.
+    const validSizesAnimatedAppAvatar: SemanticSIZES[] = 
+        [ "mini", "tiny", "small", "medium", "large", "big", "huge", "massive" ];
```

Comment:
shall we create an enum for this?

Reply:
We have an enum `SemanticSIZES `, but here we have to check the return `rest.size` is valid type of `SemanticSIZES `. The way to do this by creating an array, of all the values of `SemanticSIZES` enum.

Reply:
Instead of creating a new array, we can simply use Object.values(SemanticSIZES) to get the array of enum values

Reply:
`SemanticSIZES` is a type not an enum, my mistake.